wallet/src/lib/RapydClient.ts

```diff
@@ -0,0 +1,96 @@
+import axios from "axios";
+import * as Crypto from "crypto-js";
+import moment from "moment";
+import config from "../server/config";
+import {
+  HttpMethods,
+  IRapydClient,
+  IRapydRequestParams,
+  RequestSignature,
+} from "./types";
+
+class RapydClient implements IRapydClient {
+  post<T>({ path, body }: IRapydRequestParams<T>): Promise<T> {
+    return new Promise<T>((resolve, reject) => {
+      const { baseUrl, saltQty, accessKey, secretKey } = config.get("rapyd");
```

Comment:
I think I prefer to move away from the `config` object. It's easier just with `process.env.*`

Reply:
It'll be better.